README.md

```diff
@@ -24,24 +24,51 @@ template parameter.
 tags for the line item resource or its containing account.
 
+Two types of rules can be generated: rules for a Program Code category, or rules
+for an Owner Email category. The former assumes the tag contains a significant
+numeric code and requires additional external state, while the latter assumes that
+the entire tag value is a significant string.
+
+The order of the rules matters because processing will stop at the first match.
+For more information on rule processing, see the [AWS User Guide](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/manage-cost-categories.html).
+
+#### Program Code Category
+
+The rules generated for a Program Code category are dependent on the output of
+`lambda-mips-api` for a list of current codes, as well as the current account tags.
+
 First, this lambda will create cost-category rules that check a list of tags
 (provided as a template parameter) for every combination of program code and
-tag name, also create rules for each account with an appropriate tag, and
+tag name, then also create rules for each account with an appropriate tag, and
 finally create rules to inherit the value from a listed tag if no other rule
 matches.
 
-The order of the rules matters because processing will stop at the first match.
-For more information on rule processing, see the [AWS User Guide](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/manage-cost-categories.html).
+#### Owner Email Category
+
+Rules for an Owner Email category can also be generated from existing account
+tags.
+
+In this case, we treat the tag value an authoritative (no code deduplication),
+and so we start with inheriting tag values from tags on individual resources.
+We then fall back to rules built from account tag values for any accounts
+tagged with a listed tag.
 
 ### Components
 ![Component Diagram](docs/component.diagram.png)
 
 ### Template Parameters
+The following template parameters are used to configure Cloudformation resources
+| Template Parameter | Description | Example Value |
+| --- | --- | --- |
+| DnsName | Custom host name to use for API Gateway | cost-rules.example.com |
+| AcmCertificateArn | AWS ARN of an ACM certificate valid for `DnsName` | arn:aws:acm:us-east-1:&lt;ACCOUNT ID>:certificate/&lt;CERTIFICATE UUID> |
+
 The following template parameters are passed through as environment variables
 | Template Parameter | Environment Variable | Description | Example Value |
 | --- | --- | --- | --- |
 | ChartOfAccountsURL | ChartOfAccountsURL | URL to the chart of accounts endpoint provided by `lambda-mips-api` | https://lambda-mips-api.execute-api.amazonaws.com/accounts |
 | ProgramCodeTagList | ProgramCodeTagList | Comma-separated list of tag names that may contain program-code assignment | CostCenter,CostCenterOther |
+| OwnerEmailTagList | OwnerEmailTagList | Comma-separated list of tag names that may contain an owner email address | OwnerEmail,synapseEmail |
```

Comment:
would it make sense to change to just a general `EmailTagList` in case we use this for something other than owners?

Can we add this to example usage section?

Reply:
no, I don't think changing the name makes much sense. the tag list is named for the rule set generated with it, and the rule set has a specific purpose.

yes, I'll add an example usage section

Reply:
looks like `OwnerEmailTagList` is not in install section.  could you please add it there?

Reply:
done